src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/DefaultRazorSemanticTokenInfoServiceTest.cs

```diff
@@ -4,9 +4,13 @@
 using System;
 using System.Collections.Generic;
 using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.LanguageServer.Common;
 using Microsoft.AspNetCore.Razor.LanguageServer.Completion;
 using Microsoft.AspNetCore.Razor.LanguageServer.Semantic;
+using Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
 using Xunit;
+using LSPRange = OmniSharp.Extensions.LanguageServer.Protocol.Models.Range;
```

Comment:
```suggestion
using OmniSharpRange = OmniSharp.Extensions.LanguageServer.Protocol.Models.Range;
```
Because we use LSPRange to refer to `Microsoft.VisualStudio.LanguageServer.Protocol.Range` in a few other places.